config/settings.yml

```diff
@@ -1,10 +1,10 @@
 stacks:
   storage_root: /stacks
-  djatoka_url: 'http://localhost:8080/resolver'
   driver: djatoka
   djatoka:
     image: DjatokaImage
     info: DjatokaInfoService
+    resolver: 'http://localhost:8080/resolver'
```

Comment:
Is `resolver` the right, cross-implementation word? It seems more like a base url to me?

Reply:
Actually - that's a good question that I meant to ask this morning. Are these "resolvers" in RIIIF/Cantaloupe implementations, or should this just be `uri` and then (in this case) Djatoka is using that for it's "resolver"?

Reply:
Changed to URL.